backend/src/test/kotlin/org/loculus/backend/config/BackendSpringConfigTest.kt

```diff
@@ -0,0 +1,74 @@
+package org.loculus.backend.config
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.`is`
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.loculus.backend.controller.DEFAULT_ORGANISM
+
+class BackendSpringConfigTest {
+
+    @Test
+    fun `GIVEN an empty config THEN the it is valid`() {
+        val conf = backendConfig(emptyList(), EarliestReleaseDate(false, emptyList()))
+
+        val errors = validateEarliestReleaseDateFields(conf)
+
+        assertTrue(errors.isEmpty())
+    }
+
+    @Test
+    fun `GIVEN a config with two external fields that exist and are of type date THEN it is valid`() {
```

Comment:
Given that the test class is generally for the backend spring config, I think it's not clear from the name that the "external fields" related to the EarliestReleaseDate

Reply:
good point, fixed here: https://github.com/loculus-project/loculus/pull/3380/commits/f1f5eb49900cdd4412c9ffe0fad9fd7a22f3eaf7